packages/navigation/src/styles/Navigation.module.css

```diff
@@ -12,15 +12,20 @@
 
 .Navigation .Nav--primary {
   padding: 0 2.5rem;
```

Comment:
The background-color and box-shadow are being removed, with the expectation that consumers manage this going forward.